README.md

```diff
@@ -189,6 +196,6 @@
 
 ## 📝 License <a name="license"></a>
 
-> This project is [MIT](https://github.com/gdvl777/math_magicians/blob/events/MIT.md).
+> This project is [MIT](https://github.com/gdvl777/math_magicians/blob/develop/MIT.md).
```

Comment:
- [ ] Please note that the license link is linked to a 404 page. Kindly make sure there is no broken link in your README file. That will show your attention to detail and your professionalism.